build/buildall

```diff
@@ -171,6 +171,7 @@ case $DIST_PROFILE in
       313
       320
       321
+      330
```

Comment:
We should drop it from `snapshots?(WithDatabricks))` on L162

Reply:
good catch! updated